ADOPTERS.md

```diff
@@ -0,0 +1,14 @@
+# Adopters
+
+This page contains a non-comprehensive list of [Inspektor Gadget](https://inspektor-gadget.io/) users that we know of.
+
+## Adding your org to this list
+
+If you would like your organization or project to be added to this list, please open a PR with your name, link, and a brief description of your use case.
+
+## Inspektor Gadget adopters (in alphabetical order):
+
+| Adopter  | Description |
```

Comment:
What about https://github.com/microsoft/retina?

Reply:
https://retina.sh/docs/Metrics/plugins/Linux/dns#architecture

Reply:
Also the [tcpretrans](https://retina.sh/docs/Metrics/plugins/Linux/tcpretrans) plugin uses [Inspektor Gadget](https://github.com/microsoft/retina/blob/e4d2d3e0e8370b6f21992863f489ceefcb1fa3ee/pkg/plugin/tcpretrans/tcpretrans_linux.go#L4).